### ISSUE TITLE
Support for Flic buttons and Flic Twist

### ISSUE THREAD

**Issue by oscarb** [diyhue/diyHue#933]
### Issue does not already exist?

I have searched and found no existing issue

### Select Environment

Docker

### Home Assistant related?

No

### Description

I've received a [Flic Twist](https://flic.io/shop/flic-twist) early and I'm trying to connect to the diyHue bridge from the Flic app. 

The bridge shows up in the app but when clicking the Link button within diyhue when asked to click it in the app, nothing happens.

Not sure if this is on diyHue or the Flic end. 

I can confirm connecting Flic to a real physical Hue bridge works fine.

https://github.com/diyhue/diyHue/assets/17575446/5e56c2d3-c5d6-4b00-bcb0-f11414b0dd0c

### Errorlog:

Nothing seems to show up in the error log

### Steps to reproduce

1. Initiate paring from within the Flic app, whether it is for a Flic button or the Flic twist
2. Click Link Button in diyHue

### Please enter your operating system details here

**diyHue**
Synology NAS, DS713+, Linux 3.10.108 x86_64, DSM 7

**Flic app**
Sony Xperia 5 III, Android 13

### What DiyHue version(branch) are you using?

master (latest)

**Comment by mariusmotea** [comment]
In the video i see no requests from this device to diyhue, it catch the mDNS request locally, but most likely it continue using cloud services witch are not working with diyhue fake hub. You can try to disable the internet, maybe this will force a fallback to LAN pairing instead of cloud pairing.

**Comment by oscarb** [comment]
@mariusmotea Thanks for the tip! When I tried blocking internet access, the same happened as in the video above . 

With internet access still being off, I tried and could successfully connect the Flic hub with a Hue Bridge (square one with rounded corners).

I've also been able to successfully pair the Flic hub with [hass-emulated-hue](https://github.com/hass-emulated-hue/core), however after that even though lights show up in the Flic app nothing happens as I try to control them with the Flic Twist... 

I'd love to figure out why pairing with DiyHue won't work as I think this project has much more flexibility than for example the hass-emulated hue mentioned above. 

If you have any other tips on how I could troubleshoot this, I'm very thankful 😺
